client/dataEntry.js

```diff
@@ -139,6 +139,24 @@ function getSolveWarnings(resultId, solveIndex) {
   return warnings;
 }
 
+function updateWarningContents () {
+  let data = Template.currentData();
+  let selectedResultId = selectedResultIdReact.get();
+  let warnings = getSolveWarnings(selectedResultId, data.index);
+  let $warningIcon = template.$('.solve-time-warning-icon');
```

Comment:
After removing the one line that called it, it's not being called anywhere else. Should I remove it or leave it?